testgrid/config/config.yaml

```diff
@@ -4251,6 +4259,18 @@ dashboards:
     test_group_name: periodic-test-infra-retester
     code_search_url_template:
       url: https://github.com/kubernetes/test-infra/compare/<start-custom-0>...<end-custom-0>
+  - name: unfreeze
```

Comment:
do we want email alerts for if these fail / go stale?

Reply:
Not yet. These images don't use bootstrap yet -- waiting for the sidecar rewrite. At that time yes!